src/collision/collider/world_query.rs

```diff
@@ -0,0 +1,33 @@
+#![allow(missing_docs)]
+
+use crate::prelude::*;
+use bevy::{ecs::query::QueryData, prelude::*};
+
+/// A [`QueryData`] struct to make code handling colliders in collisions cleaner.
```

Comment:
Some usage examples would be nice

Reply:
This is intended more as an internal type, but I can make the docs a bit clearer and maybe add some basic examples